src/Features/Core/Portable/CodeCleanup/CodeCleanupOptions.cs

```diff
@@ -0,0 +1,103 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+using System.Collections.Immutable;
+using System.Composition;
+using Microsoft.CodeAnalysis.Options;
+using Microsoft.CodeAnalysis.Options.Providers;
+
+namespace Microsoft.CodeAnalysis.CodeCleanup
+{
+    internal static class CodeCleanupOptions
+    {
+        public static readonly PerLanguageOption<bool> AreCodeCleanupRulesConfigured = new PerLanguageOption<bool>(
+            nameof(CodeCleanupOptions), nameof(AreCodeCleanupRulesConfigured), defaultValue: false,
+            storageLocations: new RoamingProfileStorageLocation("TextEditor.%LANGUAGE%.Are Code Cleanup Rules Configured"));
+
+        public static readonly PerLanguageOption<bool> NeverShowCodeCleanupInfoBarAgain = new PerLanguageOption<bool>(
```

Comment:
>NeverShowCodeCleanupInfoBarAgain [](start = 55, length = 32)

Question: Which options go into [AutomationObject](http://source.roslyn.io/#Microsoft.VisualStudio.LanguageServices.CSharp/Options/AutomationObject.cs,25) and which don't? @heejaechang@outlook.com probably knows the answer..

Reply:
tracked in https://github.com/dotnet/roslyn/issues/27511